app/lib/pantry_trak/client.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module PantryTrak
+  # Creates PantryTrak client: handling of reservartions and users
+  # For user details, see pantry_trak/users.rb
+  class Client
+    def create_user(user)
+      unless user.instance_of? PantryTrak::User
```

Comment:
The `instance_of?` check feels like an anti-pattern in the ruby world. So what if the object I pass isn't a PantryTrak::User as long as it has all the necessary attributes? For example, while testing this maybe we just pass in JSON objects that have all the same attrs rather than instantiating this object. Unlike in statically typed languages, ruby tends to favor "duck-typing" https://medium.com/rubycademy/why-ruby-community-encourages-duck-typing-2e5fb529fca1